### ISSUE TITLE
What parameters use to get model 'bidaf_10.h5'?

### ISSUE THREAD

**Issue by hemavatisaboo** [ParikhKadam/bidaf-keras#12]
What parameters are used to train model 'bidaf_10.h5'? When I am trying to train the code, I am not getting that much accuracy. I used epochs=1000, steps_per_epoch=10, validation _steps=1.

**Comment by ParikhKadam** [comment]
The SQUAD v1.1 dataset contains more than 100000 question-answer pairs. You are training the model on only 10*1000=10000 pairs. That's not even a single iteration on the dataset. That's the reason you won't get accuracy.

For more analysis on SQUAD v1.1, you may look here - https://datarepository.wolframcloud.com/resources/SQuAD-v1.1

Reply if this solves your issue..

**Comment by r-baradaran** [comment]
> The SQUAD v1.1 dataset contains more than 100000 question-answer pairs. You are training the model on only 10*1000=10000 pairs. That's not even a single iteration on the dataset. That's the reason you won't get accuracy.
> 
> For more analysis on SQUAD v1.1, you may look here - https://datarepository.wolframcloud.com/resources/SQuAD-v1.1
> 
> Reply if this solves your issue..

Hello, I have the same problem. I have run  
!bidaf-keras  --do_lowercase  train --epochs=10 --batch_size=8 --save_model_per_epoch
but accuracy is very low (below 1%) and grow very slowly . what are the best parameters to train bidaf-keras to get good accuracy?
